index.html

```diff
@@ -0,0 +1,224 @@
+                        <i class="social-media-icon fa-brands fa-facebook-f"></i>
+                    </li>
+                    <li class="social-media-item">
+                        <i class="social-media-icon fa-brands fa-twitter"></i>
+                    </li>
+                </ul>
+                <ul class="login-bar">
+                    <li class="login-item">English</li>
+                    <li class="login-item">Mypage</li>
+                    <li class="login-item">Log Out</li>
+                </ul>
+            </div>
+        </div>
+        <header class="header-container">
+            <a href="#headerSection" class="logo-link">
+                <img src="assets/festival.svg" alt="Logo" class="logo-image">
+            </a>
+            <nav class="navbar">
+                <i id="menuOpenIcon" class="fa-solid fa-bars"></i>
+                <ul class="navbar-list">
+                    <li class="navbar-item">
+                        <a href="about.html" class="navbar-link active">About</a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="#program" class="navbar-link">Program</a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="#speakersSection" class="navbar-link">Chefs</a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="#partners" class="navbar-link">Partners</a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="#footer" class="navbar-link">Contact</a>
+                    </li>
+                    <li class="navbar-item ticket-item">
+                        <a href="" class="navbar-link ticket-link">Buy tickets</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+    </section>
+    <!--End-->
+
+    <!--Main Section-->
+    <main class="main-section">
+        <section class="content-container">
+            <h1 class="welcome-message">Welcome to our Food & Wine Festival!</h1>
+            <div class="title-container">
+                <h2 class="title-primary">EXPERIENCE THE <br />TASTE OF JALISCO <br />2023</h2>
+                <h2 class="title-secondary"></h2>
+                <h2 class="title-year"></h2>
+            </div>
+            <article class="features">
+                <p class="description">
+                    Join us for an unforgettable gastronomic experience beside the scenic Lake Chapala. Enjoy the
+                    finest
+                    local cuisine, wine, and culture in a festival that unites wine enthusiasts from across the
+                    globe.
+                </p>
+                <h2 class="event-date">2023.08.18 (FRI) ~ 20 (SUN)</h2>
+                <h3 class="event-location">@ Finca V&V - Vides Selectas, San Luis Soyatlan</h3>
+            </article>
+        </section>
```

Comment:
- [ ] Great job implementing the headline of the website :muscle: :sunglasses: However it not responsive, the elements overlap when reducing the screen. Kindly make sure the section is responsive on all screen for better user experience.
<video src="https://github.com/CVILLA09/capstone-project/assets/91142432/6132e767-dc63-4da3-8532-714259fb38ca" />